### DISCUSSION TITLE
Clone shimmie config/split a booru?

### DISCUSSION THREAD

**Discussion by diamondsw** [shish/shimmie2#1011] [General]
Is there any way to clone a shimmie booru (I'm using the Docker image, if it matters) with all of the settings, extensions, users, etc - but without any of the posts? I'd like to use my current shimmie install as a template of sorts for some new ideas, and while I certainly could reconfigure it all from scratch, if there is some way to just duplicate the install and delete some key file or two to reset the posts, that would be excellent.

**Comment by shish** [comment]
I can't think of a cleaner way than copying everything and then deleting the posts

For bulk post-deletion, the first thing that comes to mind is using the bulk actions extension with the CLI - exact commands would depend on your system, but something like:

```
$ docker exec -ti shimmie2 /bin/bash
# php index.php bulk-action bulk_delete "*"
```

**Reply by diamondsw** [reply]
I tried this (on a copy, of course), and got `Command "bulk-action" is not defined.`. Starting to think I'm very likely better off just rebuilding it from scratch - this is something it's clearly not designed to do, and _There May Be Dragons_.

EDIT: Ah, that's provided by an extension. Fixed.
